### DISCUSSION TITLE
Image Overwrite - Markdown Editor - Asset Manager

### DISCUSSION THREAD

**Discussion by alohawear** [requarks/wiki#6723] [Help / Questions]
Hi Folks,

We're migrating our documentation to WikiJS. It's great!

We're having trouble understanding how image overwriting works.

For example:

We're editing a page called Data Region.

We're using the markdown editor.

We click the Insert Assets button and upload an image called data_region.png.
We click Save and Close. We realize the image is wrong.
We open the page for editing again.
We click Insert Assets button again.
We upload the new data_region.png image.
Click Back to Editor
Save and Close
The image is not changed.

The upload process seems to work fine. There's no error message. But, the Added date time stamp doesn't update.

We think the images are actually stored in the progesql database, not on the file system? But there's the Administrator function to "Purge Render Cache" so maybe if we want it to upgrade quickly, we have to do that?

Thanks for any advice...
Nat

Platform: DigitalOcean
Install Method: DigitalOcean App
WikiJS Version: 2.5.29
Node Version: 16.18.0
PostgreSQL: 11.8
OS: Docker on Ubuntu

**Comment by P4mes4n** [comment]
The page might not re-render when there is no actual change in the text on the page. Have you tried changing some text an then saving the page again?
Alternatively you can force a re-render of all pages in the administration panel.

**Comment by alohawear** [comment]
Excellent suggestions. I'll check those. Thanks.

**Reply by alohawear** [reply]
No luck. I made sure to edit the text to trigger the per page re-render. I also ran the site-wide page rerender.

I think it's related to the image upload / overwrite. It shows the upload progress indicator. But after the upload is done, the Added date time stamp doesn't change.

The file size is only slightly different so maybe that has something to do with it?
